tests/integrationv2/test_client_authentication.py

```diff
@@ -1,12 +1,9 @@
 import copy
-import os
 import pytest
-import time
 
-from configuration import (available_ports, ALL_TEST_CIPHERS, ALL_TEST_CURVES,
-                           ALL_TEST_CERTS, PROTOCOLS)
+from configuration import (available_ports, ALL_TEST_CIPHERS, PROTOCOLS)
 from common import Certificates, ProviderOptions, Protocols, data_bytes
-from fixtures import managed_process
+from fixtures import managed_process # lgtm [py/unused-import]
```

Comment:
Why are all of these getting flagged as unused ? Does this check need to be disabled and relied upon in autopep8/someotherpythonlinter instead?

Reply:
I'm honestly not 100% sure. From what I've seen there isn't a direct reference to the imported `managed_process`; I'll add a note on this in the tracking issue

Reply:
For the fixture maybe, where we're not 'calling' it, but the [npn test](https://github.com/aws/s2n-tls/blob/main/tests/integrationv2/test_npn.py#L50) does and still got flagged. Annoying.